lib/discourse_theme/scaffold.rb

```diff
@@ -105,56 +23,60 @@ def self.generate(dir)
 
       is_component = UI.yes?("Is this a component?")
 
-      FileUtils.mkdir_p dir
+      if Dir.exist?(SKELETON_DIR)
+        puts `cd #{SKELETON_DIR} && git pull`
+      else
+        FileUtils.mkdir_p(SKELETON_DIR)
+        puts `git clone https://github.com/discourse/discourse-theme-skeleton #{SKELETON_DIR}`
+      end
```

Comment:
I'm not sure about this `SKELETON_DIR` strategy. There's a risk it could get out-of-date and then we'll end up throwing errors or using an out-of-date version.

So... two thoughts:

1. We should add error handling on these calls to `git`

2. Can we skip the middleman and just clone straight into the destination?

```ruby
system "git", "clone", "https://github.com/discourse/discourse-theme-skeleton", dir, exception: true
```